vocabularies/Common.xml

```diff
@@ -803,6 +803,11 @@ The minimum structure is
           <Annotation Term="Core.Description" String="Initial value, e.g. empty string, is a valid and significant value" />
         </Property>
       </ComplexType>
+      <ComplexType Name="ValueListRangeIn" BaseType="Common.ValueListParameterIn">
+        <Property Name="Operator" Type="UI.ValueListRangeOperator" DefaultValue="EQ" Nullable="false">
```

Comment:
Is this the proper way of passing an `Enum` value as `DefaultValue`? (Couldn't find other similar samples in the repo, only `true` or `false`.)

Reply:
https://github.com/SAP/odata-vocabularies/pull/375/files#r2007450478